### DISCUSSION TITLE
what is the best car

### DISCUSSION THREAD

**Discussion by ItsEnderium** [CluckCluckChicken/Magnifier#29] [Q&A]
snapchat car
xbox car
skype car
nintendo car
d##cord car
WHATSAPP CAR
**PLAYSTATION CAR**

**Comment by davwat** [comment]
dababy car

**Reply by ItsEnderium** [reply]
lil nas x car

**Comment by JackyBorderCollie** [comment]
whatsapp car

**Comment by CST1229** [comment]
[scratch car](https://scratch.mit.edu/projects/520464126/)

**Comment by brourbeinsus** [comment]
![F2AE9598-8365-4278-A281-FAAA7EE2C05D](https://user-images.githubusercontent.com/79854224/120909034-967d5400-c63e-11eb-927c-fda452098921.jpeg)

dababy car

**Comment by FunctionalMetatable** [comment]
sad car

**Comment by CST1229** [comment]
after scratch car, this car
![purple car](https://user-images.githubusercontent.com/68464103/122013330-5ea0aa00-cdbe-11eb-84b2-6b7410cc3920.png)

**Comment by davwat** [comment]
evidently the chromemobile:
![chromemobile](https://user-images.githubusercontent.com/46281840/122442218-3e2b4800-cf96-11eb-83d0-14a9de8522af.png)

**Reply by CST1229** [reply]
To the chromemobile!
